.semaphore/semaphore.yml

```diff
@@ -34,10 +35,15 @@ blocks:
         - name: "Install"
           commands:
             - cache restore
+            - tar zcvf cache.tgz .m2 target
+            - artifact push project cache.tgz
+            - rm ~/.m2/settings.xml
             - >
               MAVEN_OPTS="${MAVEN_OPTS} -Xmx3000m" ${MVN} clean install
               -q -ff ${MAVEN_SKIP} ${MAVEN_SKIP_TESTS} -T1C
             - cache store
+            - tar zcvf cache-post.tgz .m2 target
+            - artifact push project cache-post.tgz
```

Comment:
after mvn install, the artifacts needed for subsequents jobs in the build are stored in .m2 and target, I don't think this should be project level though, it needs to be specific to this workflow to avoid conflicts between concurrent builds.